### DISCUSSION TITLE
Did diesel can create sqlite database file in runtime

### DISCUSSION THREAD

**Discussion by joebnb** [diesel-rs/diesel#3720] [Q&A]
As title says,i want create a sqlite database file in runtime,this file may delete by user and recreate when some trigger trigged

did diesel have som api to handle sqlite file crate,init table etc.

same with #3129

**Comment by weiznich** [comment]
It's always hard to answer such short questions. Consider adding more details about your exact use case and what you've already tried to your question. You might also want to include information about your diesel version and the enabled feature flags.
